docs/configuration.md

```diff
@@ -1,6 +1,19 @@
 # Configuring source{d} Lookout
 
-**source{d} Lookout** is configured with the `config.yml` file, you can use the template [`config.yml.tpl`](/config.yml.tpl) to create your own. Use the `lookoutd` option `--config` to set the path to it, or use the default location at `./config.yml`.
+The behavior of **source{d} Lookout** is configured with two different configuration files:
+
+- [`config.yml`](#config.yml), to define the global configuration of the server.
+- [`.lookout.yml`](#.lookout.yml), to override the analyzer default behavior in each repository.
+
+
+# config.yml
+
+**source{d} Lookout** is configured with the `config.yml` file, you can use the template [`config.yml.tpl`](/config.yml.tpl) to create your own. Use the `lookoutd` option `--config` to set the path to it, or use the default location at `./config.yml`. The config file is read on server startup, so it is needed to be restarted to load a new configuration.
+
+The most important things you need to configure for a local installation, are:
+
+1. [Repositories](#repositories): add the URLs of the repositories to be watched.
```

Comment:
This is only the case if you are using GitHub token instead of GitHub App auth. This is getting a bit confusing, and I think we should consider separating both auth methods more clearly in the docs. If that case maybe it makes sense to leave this line as you propose, and then come back to it in a later PR.

Reply:
What if reworded this other way?
```suggestion
1. [Repositories](#repositories): define the repositories to be watched.
```
This way we only link to `Repositories` section, where it's explained the two different ways to specify the repositories to be watched.

Reply:
:+1: